components/featured-packages.js

```diff
@@ -0,0 +1,63 @@
+import { jsx, Component, Helmet } from "nano-jsx";
+import { Header } from "./header.js";
+import { Footer } from "./footer.js";
+import { getRecentPackages } from "../utils.js";
+
+export class FeaturedPackages extends Component {
+  static async fetchPackages() {
```

Comment:
this doesn't need to be static method as the result is being passed as prop [here](https://github.com/fusionstrings/jspm-packages/pull/3/files/0acec6e74c204d308a1335ae1edaf98a6ad1eb52#diff-a4c65ede64197e1a112899a68bf994485b889c4b143198bac4af53425b38406fR38) where `getRecentPackages` can be directly called.

In-turn, this can be a functional component :)

Reply:
Yup, i was trying something and left without clearing properly my bad 😅. `Suspense` works only when the comp is `hydrated` in `nano-jsx`. And so there will be collision with server rendered packages list and list loaded after hydration. So, removed it here. Will clean this up 👍